.github/labeler.yml

```
@@ -0,0 +1,67 @@
+# Common Labels:
+# Add 'oss workflows' label to any change to *.yml files within the workflows directory
+oss workflows:
+  - .github/workflows/*.yml
+
+# Add 'architecture' label to any change to *.yml files within the workflows directory
+architecture:
+  - .github/workflows/*.yml
+
+# Add 'dependencies' label to any change to package.json
+dependencies:
+  - package.json
+
+# Add '⚙️ Dev' label to any change to *.js files within the server directory
+⚙️ Dev:
+  - server/**/*.js
```

I personally find it very misleading to have a `Dev` label for the back-end when we have a separate front-end label. Please make sure, before updating the PR, to also update the label name here (you have the permissions): https://github.com/ProgramEquity/amplify/labels?q=back

```suggestion
⚙️ Back-end:
  - server/**/*.js
```

---

That makes sense to me @paramsiddharth. I have updated the label from Dev to Back-end.